### ISSUE TITLE
Need to revisit PR #1501

### ISSUE THREAD

**Issue by mrg2001** [vitmalina/w2ui#1513]
See comments https://github.com/vitmalina/w2ui/pull/1501

**Comment by Connum** [comment]
I also compared the performance before creating the pull request, and on my machine, I didn't notice any performance drop in the test/grid.html example. It is, however, a fairly powerfull system. If there are performance issues on systems with weaker specs, we'll have to see whether there are any possible improvements that can be made.

I can look into it, but I can't say how soon I will have the time to do so. Maybe it's not too bad if another pair of eyes has a look at my code! :)

**Comment by mrg2001** [comment]
Here is my observations. I could not definitively prove that the scroll is slower, but it "feels" slower on mac because of how mac works. I see in the code that once you wheel the mouse, it will scroll in set increments and not more often then once per 150ms. However, on mac when using mouse pad it diminishes user experience as it is not smooth. It jumps. So, I have disabled custom wheel scroll for normal columns and left it in tact for frozen columns. 

I also found another problem when custom scroll is enabled on normal columns. If you have many columns you cannot scroll horizontally. This is working fine when system handles the scroll.
